ios/maps/MCMapView.swift

```diff
@@ -22,8 +22,10 @@ open class MCMapView: MTKView {
     private var saveDrawable = false
     private lazy var renderToImageQueue = DispatchQueue(label: "io.openmobilemaps.renderToImagQueue", qos: .userInteractive)
 
-    private var framesToRender: UInt = 1
-    private let framesToRenderAfterInvalidate: UInt = 25
+    private var framesToRender: Int = 1
```

Comment:
Das bräuchte es wohl auch auf Android

Reply:
Umsetzung wird in feature/gl-finish-on-breaks angegangen